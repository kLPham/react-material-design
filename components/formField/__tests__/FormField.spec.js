import React from 'react';
import { shallow } from 'enzyme';
import FormField from '../index';

describe('<FormField />', () => {
    it('should render', () => {
        const formField = shallow(
            <FormField>
                <button>Click</button>
            </FormField>,
    );
        expect(formField).toMatchSnapshot();
    });
    it('should render with alignEnd', () => {
        const alignEnd = true;
        const formField = shallow(
            <FormField alignEnd={alignEnd}>
                <button>Click</button>
            </FormField>,
    );
        expect(formField).toMatchSnapshot();
    });
});
