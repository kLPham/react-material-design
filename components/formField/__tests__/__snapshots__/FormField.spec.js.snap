// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormField /> should render 1`] = `
<div
  className="mdc-form-field"
>
  <button>
    Click
  </button>
</div>
`;

exports[`<FormField /> should render with alignEnd 1`] = `
<div
  className="mdc-form-field mdc-form-field--align-end"
>
  <button>
    Click
  </button>
</div>
`;
